controllers/v1/qa_model_controller_v1.py

```diff
@@ -26,7 +27,9 @@ def create_qa_model(request: dict) -> QAModelResponse:
 def read_qa_model(id_: str) -> QAModelResponse:
     '''Controller function to get a given model based on id'''
 
-    model = qa_model_service.read_qa_model_by_id(id_)
+    #model = qa_model_service.read_qa_model_by_id(id_)
```

Comment:
Same here, we can just remove the commented line for the old method

Reply:
Controller havent tested completely, thus I comment this out first

Reply:
Same comment as above, would be worth adding a comment that it should be updated once we test.